src/utils.js

```diff
@@ -165,3 +166,42 @@ export function ensureDefinedConfig(object, requester) {
     }
   });
 }
+
+/**
+ * This function generates and returns the API config url based on the following environment variables:
+ *
+ *  MFE_CONFIG_API_PATH: API config path, this must not include the domain.
+ *  MFE_CONFIG_API_PORT(Optional): Specify the port, useful in development environments
+ *  MFE_CONFIG_API_REGEX: This regex will be used to generate the config domain based on the current domain.
+ *  APP_ID(Optional): This value makes reference to specific settings for a MFE.
+ *
+ * @returns {string}
+ */
+export function generateAPIConfigUrl() {
+  const {
```

Comment:
I think these variables should be params, in case of this function maintenance in util.js file